tests/CppUTestExt/MockParameterTest.cpp

```diff
@@ -634,6 +634,20 @@ TEST(MockParameterTest, outputParameterSucceeds)
     mock().checkExpectations();
 }
 
+TEST(MockParameterTest, unmodifiedOutputParameterSucceeds)
+{
```

Comment:
This test is copy-pasted. The retval variable doesn't make a lot of sense in this test. Could you please clean this up and also check the rest of the code that this didn't happen?

Also, remove comments please :)